client/src/components/FormField/RegisteredMFAMethodListField/RegisteredMFAMethodListField.scss

```diff
@@ -1,9 +1,40 @@
 .registered-mfa-method-list-field {
   padding: $input-padding-y 0;
+
+  &__button, &__no-methods {
```

Comment:
I think this should have a line drop after the comma, surprised sass lint doesn't pick that up